CHANGELOG.md

```diff
@@ -4,10 +4,10 @@
 * None.
 
 ### Enhancements
-* None.
+* Add support for excluding properties from an Object schema. The properties excluded are constants or properties annotated with `@Ignore` or `@Transient`. Fixes [#278](https://github.com/realm/realm-kotlin/issues/278).
```

Comment:
Not sure excluding `val`'s is a great idea. E.g. right now we are considering if primary keys should be marked `val` as they are read-only. And "proper" constants are already ignored in companion objects. 

Any strong reason to ignore them?